cmd/minishift/cmd/start.go

```diff
@@ -60,7 +60,7 @@ import (
 const (
 	commandName             = "start"
 	defaultInsecureRegistry = "172.30.0.0/16"
-	unsupportedIsoUrlFormat = "Unsupported value for iso-url. It can be an URL, file URI or one of the following short names: [b2d centos]."
+	unsupportedIsoUrlFormat = "Unsupported value for iso-url. It can be an URL, file URI or one of the following short names: [b2d centos minikube]."
```

Comment:
`b2d, centos, minikube` should be extracted into constants as `supportedISOs`.

Reply:
@budhrg I think we should handle this in an separate issue for the suggested code restructuring as I wanted to merge this asap for v1.7.0 release

Reply:
Sure :+1: